web-ui/view/github.ml

```diff
@@ -178,7 +189,12 @@ let return_link ~org ~repo ~hash =
    instead of providing a return link *)
 let list_steps ~org ~repo ~refs ~hash ~jobs ~first_step_queued_at
     ~total_run_time ?(success_msg = div []) ?(fail_msg = div [])
-    ?(return_link = div []) ?(flash_messages = []) ~csrf_token () =
+    ?(return_link = div []) ?(flash_messages = [])
+    ?(build_status : Client.State.t = Passed) ~csrf_token () =
+  (*FIXME: Fix the interface for this function so that we drop the things we are now ignoring *)
+  ignore success_msg;
+  ignore fail_msg;
+  ignore return_link;
```

Comment:
Could you open an issue on [OCaml-ci](https://github.com/ocurrent/ocaml-ci/issues/new) with an explanation and replace the FIXME message with a link to the issue? It will be easier to track FIXME and todos over time this way.
(This is a suggestion, feel free to ignore ;) )

Reply:
Will do.